Sources_v3/Controllers/ChannelController/ChannelController.swift

```diff
@@ -136,7 +136,13 @@ public class ChannelControllerGeneric<ExtraData: ExtraDataTypes>: Controller, De
     }
 
     /// The worker used to fetch the remote data and communicate with servers.
-    private lazy var worker: ChannelUpdater<ExtraData> = self.environment.channelUpdaterBuilder(
+    private lazy var updaterWorker: ChannelUpdater<ExtraData> = self.environment.channelUpdaterBuilder(
```

Comment:
I think just `updater` would be fine. The `updater` and `typingEventsSender` combo looks clear enough to me :)